docs/user_guide/queue_config.md

```diff
@@ -395,6 +395,7 @@ There is no guaranteed resource setting for the root queue.
 
 Maximum resources when configured place a hard limit on the size of all allocations that can be assigned to a queue at any point in time.
 A maximum resource can be set to 0 which makes the resource not available to the queue.
```

Comment:
change from:
> A maximum resource can be set to 0 which makes the resource not available to the queue.

to:
A single resource type within the maximum resource can be set to 0 which makes that type not available to the queue.